src/javascript/app_2/Stores/Modules/Contract/contract-store.js

```diff
@@ -131,8 +142,14 @@ export default class ContractStore extends BaseStore {
 
         this.contract_info = response.proposal_open_contract;
 
+        // Set contract symbol if trade_symbol and contract_symbol don't match
         if (this.root_store.modules.trade.symbol !== this.contract_info.underlying) {
-            this.root_store.modules.trade.updateSymbol(this.contract_info.underlying);
+            this.root_store.modules.trade.onChange({
```

Comment:
Think we could put all this in `updateSymbol`?

Reply:
`updateSymbol` is removed from `trade_store` in this PR. We should handle it similar to other onChange events since it's getting the `target`'s `name` and `value`.

Reply:
Oh I mean we could put the `onChange` into the `updateSymbol` to provide an abstraction for updating symbol then we don't need all of:

```
this.root_store.modules.trade.onChange({
                target: {
                    name : 'symbol',
                    value: this.trade_chart_symbol,
                },
            });
```
+ if symbol updating ever changes in the future there is only one place that needs to be updated since we are changing the symbol in quite a few places now.